Branch-SDK/Branch-SDK/BranchGetAppListRequest.h

```diff
@@ -1,16 +0,0 @@
-//
```

Comment:
The majority of the idea w/ this PR was to add error handling, which in turn allows us to safely remove requests w/o fear of pending requests in the queue biting us. 

However, this made me realize that I'm wrapping the *entire* block in a try -- it might be better to have a try around *each* so that one request being discarded doesn't result in the whole block of requests being thrown out.